### ISSUE TITLE
VxTextField not call onChanged when clear of the textfield.

### ISSUE THREAD

**Issue by shinewanna** [iampawan/VelocityX#81]
```dart
VxTextField(
                  controller: _searchTextCtrl,
                  hint: 'Search',
                  borderType: VxTextFieldBorderType.roundLine,
                  fillColor: AppColor.dark,
                  onChanged: _appsCtrl.query,
                ),
```
Reproduce the issue:
Type on your VxTextField and click clear from right side of the textfield , then check the onChanged will not call.

**Comment by iampawan** [comment]
Have you added any listener for that?

**Comment by shinewanna** [comment]
No bro.

**Comment by iampawan** [comment]
Fixed in v3.0.0
